### ISSUE TITLE
export as  - Parsing error: Unexpected token as

### ISSUE THREAD

**Issue by ischyron** [eslint/eslint#6092]
**What version of ESLint are you using?**
    2.8.0

**What parser (default, Babel-ESLint, etc.) are you using?**
    "babel-eslint": "^6.0.2"

**Please show your full configuration:**
```
{
    "extends": "airbnb",
    "env" : {
      "jasmine" : true
    },
    parserOptions: {
      "ecmaVersion": 6,
      "ecmaFeatures": {
        "jsx": true,
        "modules": true,
        "experimentalObjectRestSpread": true
      },
    },
    "rules" : {
      "object-curly-spacing" : 0,
      "comma-dangle" : 0,
      "func-names" : 0,
      "max-len": ["error", 150, 2],
      "space-before-function-paren" : 0,
      "react/jsx-indent-props" : 0,
      "react/jsx-closing-bracket-location" : 0,
      "indent" : 0,
      "quotes" : 0,
      "padded-blocks" : 0,
      'no-param-reassign': [2, { 'props': false }]
    }
}
```

**What did you do? Please include the actual source code causing the issue.**
```
index.js:
export * as renderers from './renderers';

```
**What did you expect to happen?**
valid ES6 to pass

**What actually happened? Please include the actual, raw output from ESLint.**
```
/src/index.js
  1:10  error  Parsing error: Unexpected token as

```

**Comment by michaelficarra** [comment]
That's not ES6.

**Comment by platinumazure** [comment]
@michaelficarra Is there a correct equivalent syntax available in this case?

**Comment by michaelficarra** [comment]
@platinumazure There's [a stage 1 proposal](https://github.com/leebyron/ecmascript-export-ns-from) for the syntax. The equivalent is

```js
import * as ns from "module";
export {ns};
```

**Comment by kaicataldo** [comment]
I'm going to go ahead and close this, since we don't support stage 1 proposals in Espree (ESLint's default parser). It looks like you're using babel-eslint, though, right? babel-eslint uses its own parser, and you should be able to enable the transforms you want in your .babelrc. This transform might be of interest to you! https://babeljs.io/docs/plugins/transform-export-extensions/

**Comment by ischyron** [comment]
thanks all! my bad.  Not an experimental es6 feature then.
